frontend/appflowy_flutter/lib/mobile/presentation/home/recent_folder/mobile_recent_view.dart

```diff
@@ -125,16 +125,13 @@ class _MobileRecentViewState extends State<MobileRecentView> {
               right: 0,
               child: Container(
                 height: widget.height / 2.0,
-                width: double.infinity,
-                padding: const EdgeInsets.only(
-                  left: 8.0,
-                  top: 14.0,
-                  right: 8.0,
-                ),
-                child: FlowyText(
+                width: widget.height,
+                padding: const EdgeInsets.fromLTRB(8, 16, 8, 2),
+                child: AutoSizeText(
```

Comment:
I used to use this package long ago, when I first started developing in Flutter back in 2021.

AutoSizeText is quite bad, it can cause performance issue with overreliance, also, for this one simple problem, can we really not find a more suitable solution than to bring in this package?

Reply:
I mean I locked it down to only try two font sizes so I don't think it will be that bad. I've switched to another version with IntrinsicHeight.